powersimdata/tests/mock_input_data.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import pandas as pd
+
+from powersimdata.input.grid import Grid
+
+
+class MockInputData:
```

Comment:
It would be great if we can have some doc strings here given it is now in a user facing `tests` module.

Reply:
Sure, added docs and tests!